interns/samuel-hudson-emig/journals/index.html

```diff
@@ -12,5 +12,18 @@ <h1>Journals</h1>
     width="800px"
     height="500px">
   </object>
+
+ <object
+    data="./HudsonE Week 2 Journal Entries - HS internship program.pdf"
+    width="800px"
+    height="500px">
+  </object>
+ 
+ <object
+    data="./HudsonE Week 3 Journal Entries - HS internship program.pdf"
```

Comment:
Is this the exact name of the file that you have saved on your computer? "HudsonE Week 2 Journal Entries - HS internship program.pdf" and "HudsonE Week 3 Journal Entries - HS internship program.pdf"

Please add a screen capture to confirm

Reply:
@HudsonE7 ^